src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,111 @@
+import classNames from 'classnames';
+import React, { useContext, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoContext } from '../../context/TodoContext';
+import { deleteTodos, updateTodos } from '../../api/todos';
+import { Error } from '../../types/Error';
+import { EditForm } from '../EditForm';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.